### PR TITLE
Add readme for YOLOv3 integration

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

**Review by bfineran — DISMISSED** [review]

File: integrations/ultralytics-yolov3/README.md [hunk 1]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+- Blog: [YOLOv3 on CPUs: Sparsifying to Achieve GPU-Level Performance](https://neuralmagic.com/blog/benchmark-yolov3-on-cpus-with-deepsparse/)
+- Inference Example: [DeepSparse YOLOv3 Example](https://github.com/neuralmagic/deepsparse)
+
+## Tutorials
+
+- Coming soon!
+
+## Installation
+
+To begin using this integration, run the following command in the root directory:
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:41]: @markurtz and @bfineran is it obvious in the root directory of [what]? sparseml?

File: integrations/ultralytics-yolov3/README.md [hunk 2]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+
+| Model                       | Dataset | mAP@50 | Description                                                    | SparseZoo Stub                                                                   |
+|-----------------------------|---------|--------|----------------------------------------------------------------|----------------------------------------------------------------------------------|
+| YOLOv3-SPP Pruned Quantized | COCO    | 60.4   | 83% sparse and INT8 quantized model with LeakyReLU activations | zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/pruned_quant-aggressive_94 |
+| YOLOv3-SPP Pruned           | COCO    | 62.4   | 88% sparse model with LeakyReLU activations                    | zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/pruned-aggressive_97       |
+| YOLOv3-SPP                  | COCO    | 64.2   | Baseline model with LeakyReLU activations                      | zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/base-none                  |
+
+Web tables are available for complete lists for all [models](https://sparsezoo.neuralmagic.com/tables/models/cv/detection?repo=ultralytics) and
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:80]: What are "web tables" @markurtz or @bfineran?
Is it simply "Complete lists are available online for alll..."

File: integrations/ultralytics-yolov3/README.md [hunk 3]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+```python
+from sparsezoo import Zoo
+
+recipe = Zoo.load_recipe_from_stub("zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/pruned_quant-aggressive_94/original")
+print(recipe)
+```
+
+### Structure
+
+The following table lays out the root level files and folders along with a description for each.
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:101]:
```suggestion
The following table lays out the root-level files and folders along with a description for each.
```

File: integrations/ultralytics-yolov3/README.md [hunk 4]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+
+| Folder/File Name     | Description                                                                                                           |
+|----------------------|-----------------------------------------------------------------------------------------------------------------------|
+| data                 | Folder containing the hyperparameters to use and files to use for training on data.                                   |
+| models               | Folder containing model architecture definitions along with any downloaded checkpoints from the SparseZoo.            |
+| recipes              | Folder containing typical recipes for sparsifying YOLOv3 models along with any downloaded recipes from the SparseZoo. |
+| tutorials            | Folder containing Tutorial walkthroughs for how to sparsify YOLOv3 models using recipes.                              |
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:108]:
```suggestion
| tutorials            | Folder containing tutorial walkthroughs for how to sparsify YOLOv3 models using recipes.                              |
```

File: integrations/ultralytics-yolov3/README.md [hunk 5]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+
+recipe = Zoo.load_recipe_from_stub("zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/pruned_quant-aggressive_94/original")
+print(recipe)
+```
+
+### Structure
+
+The following table lays out the root level files and folders along with a description for each.
+
+| Folder/File Name     | Description                                                                                                           |
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:103]:
```suggestion
| Folder/File Name     | Folder Description                                                                                                           |
```

File: integrations/ultralytics-yolov3/README.md [hunk 6]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+|----------------------|-----------------------------------------------------------------------------------------------------------------------|
+| data                 | Folder containing the hyperparameters to use and files to use for training on data.                                   |
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:105]:
```suggestion
| data                 | The hyperparameters to use and files to use for training on data.                                   |
```

File: integrations/ultralytics-yolov3/README.md [hunk 7]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+| models               | Folder containing model architecture definitions along with any downloaded checkpoints from the SparseZoo.            |
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:106]:
```suggestion
| models               | Model architecture definitions along with any downloaded checkpoints from the SparseZoo.            |
```

File: integrations/ultralytics-yolov3/README.md [hunk 8]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+| recipes              | Folder containing typical recipes for sparsifying YOLOv3 models along with any downloaded recipes from the SparseZoo. |
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:107]:
```suggestion
| recipes              | Typical recipes for sparsifying YOLOv3 models along with any downloaded recipes from the SparseZoo. |
```

File: integrations/ultralytics-yolov3/README.md [hunk 9]
```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
...
+After sparsifying a model, the [`export.py` script](https://github.com/neuralmagic/yolov3/blob/master/models/export.py) 
+converts the model into deployment formats such as [ONNX](https://onnx.ai/).
+The export process is modified such that the quantized and pruned models are corrected and folded properly.
+
+For example, the following command can be run from within the yolov3 repository folder to export a trained/sparsified model's checkpoint:
+```bash
+python models/export.py --weights PATH/TO/weights.pt --img-size 512 512
+```
+
+The DeepSparse Inference Engine accepts ONNX formats and is engineered to significantly speed up inference on CPUs for the sparsified models from this integration.
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/README.md:124]:
```suggestion
The DeepSparse Engine accepts ONNX formats and is engineered to significantly speed up inference on CPUs for the sparsified models from this integration.
```

File: integrations/ultralytics-yolov3/README.md (hunk 1 above)
Comment (markurtz) [reply integrations/ultralytics-yolov3/README.md:41]: root directory of the integration, rewording to put integration at the end

File: integrations/ultralytics-yolov3/README.md (hunk 2 above)
Comment (markurtz) [reply integrations/ultralytics-yolov3/README.md:80]: yes, updated to clarify with that suggestion

**Comment by markurtz** [comment]
@jeanniefinks @bfineran updated from review now

**Review by bfineran — APPROVED** [review]

**Review by jeanniefinks — APPROVED** [review]

**Pull request merged**